powerVCF.psm1

```diff
@@ -1248,7 +1271,8 @@ Function Validate-CommissionHostSpec {
 		Remove-TypeData System.Array 
 		}
     try { 
-	$response = Invoke-RestMethod -Method POST -URI $uri -ContentType application/json -headers $headers -body $body
+    $response = Invoke-RestMethod -Method POST -URI $uri -ContentType application/json -headers $headers -body $body
+    return $response
```

Comment:
in order to check the validation task from Commission-VCFHost this return was needed